.github/workflows/integration-tests.yml

```
@@ -23,6 +23,17 @@ jobs:
         uses: arduino/setup-protoc@v3
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install mold
+        uses: rui314/setup-mold@v1
+      - name: Enable mold on Linux
+        run: |
+          if [[ "${{ matrix.os }}" == "ubuntu-latest" ]]; then
+            mkdir .cargo
+            echo "[target.x86_64-unknown-linux-gnu]" >> .cargo/config.toml
+            echo "linker = \"clang\"" >> .cargo/config.toml
+            echo "rustflags = [\"-C\", \"link-arg=-fuse-ld=/usr/local/bin/mold\"]" >> .cargo/config.toml
+          fi
+        shell: bash
```

Suggestion to tidy up:

```suggestion
      - name: Install mold
        if: matrix.os == 'ubuntu-latest'
        uses: rui314/setup-mold@v1
      - name: Enable mold
        if: matrix.os == 'ubuntu-latest'
        run: |
          mkdir .cargo
          cat > .cargo/config.toml << EOF
          [target.x86_64-unknown-linux-gnu]
          linker = "clang"
          rustflags = ["-C", "link-arg=-fuse-ld=/usr/local/bin/mold"]
          EOF
```